python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+## @file BenderBox.py
+#  Bender application for Ganga  (CMAKE-based version)
+#  Actually there are three applications under the single hat
+#  - BenderModule : classic Bender application: use a full power of bender 
+#  - BenderRun    : execute scripts in the context of ``bender'' environment
+#  - OstapRun     : execute scripts in the context of ``ostap'' environment
+#  @author Vladimir ROMANOVSKY Vladimir.Romanovskiy@cern.ch
+#  @author Vanya BELYAEV  Ivan.Belyaev@itep.ru       
+# =============================================================================
+"""Bender application for Ganga  (CMAKE-based version)
+Actually there are three applications under the single hat
+- BenderModule : classic Bender application: use a full power of bender 
+- BenderScript : execute scripts in the context of ``bender'' environment
+- Ostap : execute scripts in the context of ``ostap'' environment
+"""
+# =============================================================================
+import os
+from   Ganga.GPIDev.Lib.File.File      import ShareDir,File
+from   Ganga.GPIDev.Lib.File.LocalFile import LocalFile
+from   Ganga.GPIDev.Schema             import Schema, Version, SimpleItem, GangaFileItem, FileItem, ComponentItem
+from   Ganga.GPIDev.Base               import GangaObject
+
+# =============================================================================
+## @class Bender_module
+#  Helper class to define the main properties of Bender applicatiom
+#  - the name of Bender module to run
+#  - dictionary of parameters to be forwarder to <code>configure</code> method
+#  - number of event to process
+#
+#  Usage:
+#  @code 
+#  j.application = BenderBox ( tool = BenderModule ( module = 'the_path/the_module.py' , events = 1000 ) )
+#  @endcode 
+#  @author Vladimir ROMANOVSKY Vladimir.Romanovskiy@cern.ch
+#  @author Vanya BELYAEV  Ivan.Belyaev@itep.ru       
+class BenderModule(GangaObject):
```

Comment:
Is this ever used anywhere over being used inside `BenderBox`? Why no just directly implement this functionality there rather than having another class (I may have misunderstood the working here).